x-pack/plugins/alerting/server/lib/rule_execution_status.ts

```diff
@@ -14,6 +14,7 @@ import { AlertExecutionStatuses } from '../../common';
 export function executionStatusFromState(state: RuleTaskState): AlertExecutionStatus {
   const alertIds = Object.keys(state.alertInstances ?? {});
   return {
+    numberOfTriggeredActions: state.triggeredActions?.length ?? 0,
```

Comment:
ah, I guess this is why the triggered actions got added to the task state - because we get the other status bits from it?  Looks like it's only called from here, and I'm guessing we could find some way to thread the value through the other calls, to avoid having to add it to the task state.

https://github.com/elastic/kibana/blob/c3866c40322317143b660067897c79f12e0a4311/x-pack/plugins/alerting/server/task_runner/task_runner.ts#L656-L660